.linkcheckerrc

```diff
@@ -0,0 +1,3 @@
+[filtering]
+ignorewarningsforurls=
+  ^https://stackoverflow\.com ^http-rate-limited$

```

Comment:
Another file at the root just for the docs? :(
Can this be avoided? I see that you pass the path of the config file on the command line, maybe it can be moved to `docs`?

Reply:
Nevermind that. Since I have another PR to move everything docs-related to `docs`, I'll take care of it in my own PR after merging yours.